src/components/cells/cell-container.jsx

```diff
@@ -36,10 +45,20 @@ export class CellContainerUnconnected extends React.Component {
     this.editor = React.createRef()
   }
 
-  handleCellClick = () => {
-    const scrollToCell = false
-    if (!this.props.selected) {
-      this.props.selectCell(this.props.cellId, scrollToCell)
+  handleCellClick = (event) => {
+    if (!event.target.classList.contains('CodeMirror-line')) {
```

Comment:
is this conditional meant to prevent clicks in the text editor from enabling highlighting? if so, it's not really working for me (clicks in the text editor are still enabling the highlighting). if that is the intent, we could just limit highlight clicks to the  cell header (since that would be pretty much the only click area left). You could  create a new handler `handleCellHeader` and attach it to the cell-header div.

But i'm not 100% sure what your objective was here, so that may not work depending on what you were trying to accomplish